src/startpage/startpage.cpp

```diff
@@ -14,9 +14,9 @@ StartPage::StartPage(QWidget *parent) :
     setWindowTitle("Start Page");
 
     int maxButtonsWidth = 200;
```

Comment:
Use const if maxButtonsWidth is not changing by other function.

Reply:
done